test/integration/verify/controls/aws_transit_gateway_multicast_domain_associations.rb

```diff
@@ -0,0 +1,19 @@
+control 'aws_transit_gateway_multicast_domain_associations' do
+  impact 1.0
+  title 'Gets information about the associations for the transit gateway multicast domain.'
+
+  describe aws_transit_gateway_multicast_domain_associations(transit_gateway_multicast_domain_id: "") do
+    it { should exist }
+  end
+
+  describe aws_transit_gateway_multicast_domain_associations(transit_gateway_multicast_domain_id: "") do
+    its('transit_gateway_attachment_ids') { should include '' }
+    its('resource_ids') { should include '' }
+    its('resource_types') { should include 'vpc' }
+    its('resource_owner_ids') { should include '' }
+  end
```

Comment:
why are we not testing anything here ?

Reply:
I skipped it as the testing has to be done manually.

Reply:
Done with the dummy values